src/main/java/org/spongepowered/api/text/Text.java

```diff
@@ -77,6 +80,10 @@
  */
 public abstract class Text implements TextRepresentable {
 
+    static {
+        TypeSerializers.getDefaultSerializers().registerType(TypeToken.of(Text.class), new TextConfigSerializer());
```

Comment:
@windy1 I don't see the `DataSerializable` changes yet, unless you changed your mind?